File: Logic Building Assignments/Logic Building Assignment 2/Assignment2_2.c
//  2. Accept one number from user and print that number of * on screen.

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    //Updator
    while(iNo > iCnt)
    {
        printf("*\t");
        iNo--;
    }
}

int main()
{
    int iValue = 0;

    printf("Enter your number : ");
    scanf("%d",&iValue);

    Display(iValue);
    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 2>gcc Assignment2_2.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 2>a
    Enter your number : 5
    *       *       *       *       *

*/

File: Python Assignments/Python Assignment 2/Assignment2_2.py
#   2. Write a program which accept one number and display below pattern.

def Pattern(No):
    for i in range(No):
        for j in range(No):
            print("*", end = "\t")
        print()

def main():
    Number = int(input("Enter your number : "))
    Pattern(Number)

if __name__ == "__main__":
    main()

#   Output:
#   D:\Python\Marvellous Python\Python Assignment 2>python Assignment2_2.py
#   Enter your number : 5
#   *       *       *       *       *
#   *       *       *       *       *
#   *       *       *       *       *
#   *       *       *       *       *
#   *       *       *       *       *